### DISCUSSION TITLE
[A-L] (16 Gennaio 2023) Esercizio 8

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#371] [Esami A-L]
La seguente formula proposizionale in CNF è soddisfacibile?

$$
    \Large \\{\ \\{\ p,\ \neg q,\ r,\ s\ \\},\ \\{\ \neg p,\ q,\ \neg r\ \\},\ \\{\ \neg r,\ s\ \\},\ \\{\ \neg q,\ r\ \\},\ \\{\ p,\ \neg s\ \\}\ \\}
$$

Se si risponde SI definire un assegnamento che la soddisfa, se si risponde NO dimostrare l’insoddisfacibilità usando la regola di Risoluzione.

**Comment by cek753** [comment]
SAT, p=0 q=0 r=0 s=0
